### ISSUE TITLE
CSS: Status Menu Animations

### ISSUE THREAD

**Issue by mindreeper2420** [fabric8-ui/fabric8-ux#838]
Update the status menu to animate the background color based on visual designs.

https://redhat.invisionapp.com/share/ZFEP5QKXA#/screens/266500223

**Comment by mindreeper2420** [comment]
Codepen: https://codepen.io/mindreeper2420/pen/XVGoyX

**Comment by mindreeper2420** [comment]
Code updated with new colors, icons and status identifiers, per design. At this point, only one element (jenkins) is checked for a status, so all other items are fixed to the "ok" state. Code can be found on a [branch of fabric8-ui](https://github.com/mindreeper2420/fabric8-ui/tree/osio-448).

Another story will need to be added to update to allow for dynamic switching of the icon in the navbar based on the lowest status (error, warning, ok).

**Comment by mindreeper2420** [comment]
follow up required for implementation into UI

**Comment by mindreeper2420** [comment]
@catrobson has it been decided whether or not the status drop down is being removed?

**Comment by joshuawilson** [comment]
Engineering recommends dropping it for now. Need PM/UX confirmation.

**Comment by catrobson** [comment]
Created an issue in Openshift.io around this - waiting on PM confirmation to remove.  https://github.com/openshiftio/openshift.io/issues/2859

**Comment by AdamJ** [comment]
Status drop-down removed from OSIO.
